src/reanimated2/Colors.ts

```diff
@@ -716,13 +718,17 @@ const getInterpolateCacheRGBA = (
     }
   }
   const newCache = { r, g, b, a };
-  const overrideHash = hashOrderRGBA[curentHashIndexRGBA];
+  const overrideHash = hashOrderRGBA.value[curentHashIndexRGBA.value];
   if (overrideHash) {
     delete interpolateCacheRGBA[overrideHash];
   }
-  interpolateCacheRGBA[hash] = newCache;
-  hashOrderRGBA[curentHashIndexRGBA] = hash;
-  curentHashIndexRGBA = (curentHashIndexRGBA + 1) % BUFFER_SIZE;
+  const mutableCache = {
+    ...interpolateCacheRGBA.value,
```

Comment:
It looks like here we're unpacking the whole object (effectively cloning it) but previously we were just assigning a single value to a single key.

Reply:
I agree but this object is unfortunately frozen and this is the only way to modify it.